### PR TITLE
Headers

### PR DESCRIPTION
**Pull request by jyoung488**
Added response headers

### CODE REVIEW

**Comment by jyoung488** [comment]
Addresses #38

File: lib/hello_sign/version.rb [hunk 1]
```diff
@@ -23,5 +23,5 @@
 #
 
 module HelloSign
-  VERSION = '3.6.4'
```
Comment (michaelnlindsay) [inline lib/hello_sign/version.rb:26]: 3.7.0 for proper symantic versioning

File: lib/hello_sign/resource/base_resource.rb [hunk 2]
```diff
@@ -40,8 +42,9 @@ class BaseResource
       #
       # @return [HelloSign::Resource::BaseResource] a new BaseResource
       def initialize(hash, key=nil)
-        @raw_data = key ? hash[key] : hash
-        @warnings = hash['warnings'] ? hash['warnings'] : nil
+        @headers = hash[:headers]
+        @raw_data = key ? hash[:body][key] : hash
+        @warnings = hash['warnings'] ? hash[:body]['warnings'] : nil
```
Comment (michaelnlindsay) [inline lib/hello_sign/resource/base_resource.rb]: Shouldn't this be: 
@warnings = hash**[:body]**['warnings'] ? hash[:body]['warnings'] : nil

Comment (michaelnlindsay) [reply lib/hello_sign/resource/base_resource.rb]: may want to add a test for warnings as well

**Review by michaelnlindsay — CHANGES_REQUESTED** [review]

**Comment by jyoung488** [comment]
@michaelnlindsay I finally got around to implementing this! Would you mind reviewing again for the headers and warnings?

**Review by michaelnlindsay — APPROVED** [review]
looks great!

**Pull request merged**